examples/run_examples.py

```diff
@@ -1,9 +1,25 @@
+# interactive=False
+# import subprocess
+# import os
+# import matplotlib
```

Comment:
sure, I'll say this is a dummy run_examples file that will be used in a later stage